packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Style the self-hosted Sign-In Widget
+---
+
+This section discusses the customization options that you have when you are self-hosting the Sign-In Widget.
+
+> **Note:** You can try all of these customizations yourself using our [Live Widget](https://developer.okta.com/live-widget/).
+
+## Initial sign-in page
+
+You can modify the look of the initial sign-in page using parameters in the `config` section of the main Widget initialization block.
```

Comment:
Does Widget need to be capitalized here?

Reply:
yes - it's a thing with and without Okta Sign-In in front of it.